ps-menu.psm1

```diff
@@ -34,14 +34,17 @@ function Toggle-Selection {
 	$result
 }
 
-function Menu {
-    param ([array]$menuItems, [switch]$ReturnIndex=$false, [switch]$Multiselect)
+function New-Menu {
```

Comment:
It's a breaking change which I would prefer to avoid. You can add an "alias" like this so that both names are supported (backward compatibility - otherwise people will hate it after updating the package)

Reply:
Added an alias.